src/css/coa.scss

```diff
@@ -26,7 +26,7 @@
 @import 'css/modules/Recollect';
 @import 'css/modules/HtmlFromAdmin';
 @import 'css/modules/StaticMap';
-@import 'css/modules/ContactDetails';
+@import 'css/modules/Contact';
```

Comment:
Renamed so this sass file could be pertinent to all styles for components in the Contact folder.